stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/gallery/AttachmentGalleryPageFragment.kt

```diff
@@ -6,48 +6,40 @@ import android.view.View
 import android.view.ViewGroup
 import androidx.fragment.app.Fragment
 import com.getstream.sdk.chat.images.load
-import com.getstream.sdk.chat.utils.extensions.inflater
 import io.getstream.chat.android.ui.databinding.StreamUiItemImageGalleryBinding
 
-private const val IMAGE_ID = "IMAGE_ID"
-
-internal class ImageSlidePageFragment : Fragment() {
-
+internal class AttachmentGalleryPageFragment : Fragment() {
     private lateinit var binding: StreamUiItemImageGalleryBinding
+    private val imageUrl: String by lazy { requireArguments().getString(ARG_IMAGE_URL)!! }
```

Comment:
Isn't it a problem to use `!!`? Perhaps something 
```
requireArguments().getString(ARG_IMAGE_URL) ?: throw(IllegalStateExpeption("I don't like this!!")) 
```
Would be better (probably witha different message hahaha)

Reply:
Done